src/main.js

```diff
@@ -337,58 +134,15 @@ Apify.main(async () => {
         },
         proxyConfiguration: proxyConfig,
         preNavigationHooks: [async ({ request, page }, gotoOptions) => {
-            const userAgent = headerGenerator.getHeaders()['user-agent'];
+            /** @type {any} */
+            const headers = headerGenerator.getHeaders();
+            const userAgent = headers['user-agent'];
             log.debug(`User-agent: ${userAgent}`);
 
             await page.setUserAgent(userAgent);
 
             await puppeteer.blockRequests(page, {
-                urlPatterns: [
-                    '.gif',
-                    '.webp',
-                    '.jpeg',
-                    '.jpg',
-                    '.png',
-                    '.ttf',
-                    '.css.map',
-                    'www.googletagmanager.com',
-                    'www.googletagservices.com',
-                    'www.googleadservices.com',
-                    'www.google-analytics.com',
-                    'sb.scorecardresearch.com',
-                    'cdn.ampproject.org',
-                    'doubleclick.net',
-                    'pagead2.googlesyndication.com',
-                    'amazon-adsystem.com',
-                    'tpc.googlesyndication.com',
-                    'googleads.g.doubleclick.net',
-                    'pxl.jivox.com',
-                    'ib.adnxs.com',
-                    'static.ads-twitter.com',
-                    'bat.bing.com',
-                    'px-cloud.net',
-                    'fonts.gstatic.com',
-                    'tiqcdn.com',
-                    'fonts.googleapis.com',
-                    'photos.zillowstatic.com',
-                    'survata.com',
-                    'zg-api.com',
-                    'accounts.google.com',
-                    'casalemedia.com',
-                    'adsystem.com',
-                    '/collector',
```

Comment:
Nice! I guess this is the browser fingerprint collector. It is amazing that it is enough to block it 👍

Reply:
An interesting finding 👍 Again, not my credit, just moved it from `main` to `constants` 🙂